addons/web_editor/views/snippets.xml

```diff
@@ -749,7 +749,7 @@
         </we-button-group>
     </div>
 
-    <div data-selector="span.fa, i.fa, img" data-exclude="[data-oe-type='image'] > img, [data-oe-xpath]" class="o_we_image_options">
+    <div data-js="AlignImage" data-selector="span.fa, i.fa, img" data-exclude="[data-oe-type='image'] > img, [data-oe-xpath]" class="o_we_image_options">
```

Comment:
Would need to patch `_computeSnippetTemplates` for this to have effect in existing databases